pages/common/dnsx.md

```diff
@@ -0,0 +1,37 @@
+# dnsx
+
+> A fast and multi-purpose DNS toolkit to run multiple DNS queries.
+> Note: input to `dnsx` needs to be passed through `stdin` (pipe `|`) in some cases.
```

Comment:
```suggestion
> See also: `dig`, `dog`, `dnstracer`.
```

Reply:
The first time I used `dnsx`, I found it very confusing because it did not behave like a usual CLI tool. Arguments needed to be passed via a pipe of through a switch. So the `Note` line IMO is more important than `See also`.

Reply:
Sorry, I removed it accidentally.